sdk/monitor/azure-monitor-opentelemetry-exporter/pom.xml

```diff
@@ -53,12 +53,12 @@
     <dependency>
       <groupId>io.opentelemetry</groupId>
       <artifactId>opentelemetry-api</artifactId>
-      <version>1.0.0</version> <!-- {x-version-update;io.opentelemetry:opentelemetry-api;external_dependency} -->
+      <version>1.7.0</version> <!-- {x-version-update;io.opentelemetry:opentelemetry-api;external_dependency} -->
```

Comment:
These versions shouldn't be manually updated. It should be updated in the [external_dependencies](https://github.com/Azure/azure-sdk-for-java/blob/main/eng/versioning/external_dependencies.txt#L176-L177) file and run the [Python script](https://github.com/Azure/azure-sdk-for-java/blob/main/eng/versioning/update_versions.py#L28) to update the versions in this file.